ibm/resource_ibm_scc_account_settings.go

```diff
@@ -0,0 +1,124 @@
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Used to determine when to delete data in an inactive location.",
+			},
+		},
+	}
+}
+
+func resourceIbmSccAccountSettingsRead(context context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	adminServiceClient, err := meta.(ClientSession).AdminServiceApiV1()
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	getAccountSettingsOption := &adminserviceapiv1.GetSettingsOptions{}
+
+	userDetails, err := meta.(ClientSession).BluemixUserDetails()
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	accountID := userDetails.userAccount
+
+	getAccountSettingsOption.SetAccountID(accountID)
+
+	accountSettings, response, err := adminServiceClient.GetSettingsWithContext(context, getAccountSettingsOption)
+	if err != nil {
+		if response != nil && response.StatusCode == 404 {
+			return nil
+		}
+		log.Printf("[DEBUG] GetSettingsWithContext failed %s\n%s", err, response)
+		return diag.FromErr(fmt.Errorf("GetSettingsWithContext failed %s\n%s", err, response))
+	}
+
+	d.SetId(*accountSettings.Location.ID)
+
+	return nil
+
+}
+
+func resourceIbmSccAccountSettingsUpdate(context context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	adminServiceClient, err := meta.(ClientSession).AdminServiceApiV1()
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	userDetails, err := meta.(ClientSession).BluemixUserDetails()
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	accountID := userDetails.userAccount
+
+	locationID := d.Get("location_id").(string)
+	updateAccountSettingsOption := &adminserviceapiv1.PatchAccountSettingsOptions{}
+	updateAccountSettingsOption.SetAccountID(accountID)
+	updateAccountSettingsOption.SetLocation(&adminserviceapiv1.LocationID{
+		ID: core.StringPtr(locationID),
+	})
+
+	_, response, err := adminServiceClient.PatchAccountSettingsWithContext(context, updateAccountSettingsOption)
+	if err != nil {
+		log.Printf("[DEBUG] PatchAccountSettingsWithContext failed %s\n%s", err, response)
+		return diag.FromErr(fmt.Errorf("PatchAccountSettingsWithContext failed %s\n%s", err, response))
+	}
+
+	return resourceIbmSccAccountSettingsRead(context, d, meta)
+}
+
+func resourceIBMSccAccountSettingsValidator() *ResourceValidator {
+	validateSchema := make([]ValidateSchema, 2)
+	validateSchema = append(validateSchema,
+		ValidateSchema{
+			Identifier:                 "location_id",
+			ValidateFunctionIdentifier: ValidateAllowedStringValue,
+			Type:                       TypeString,
+			Required:                   true,
+			AllowedValues:              "us, eu, uk",
+		},
+	)
+
+	resourceValidator := ResourceValidator{ResourceName: "ibm_scc_account_settings", Schema: validateSchema}
+	return &resourceValidator
+}
+
+func resourceIbmSccAccountSettingsDelete(context context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	return resourceIbmSccAccountSettingsRead(context, d, meta)
```

Comment:
If we don't have any API to delete just retrun nil (no need again to make call to Read)